src/plugins/data_source/server/index.ts

```diff
@@ -11,6 +11,7 @@ export const config: PluginConfigDescriptor<DataSourcePluginConfigType> = {
   exposeToBrowser: {
     enabled: true,
     hideLocalCluster: true,
+    authTypes: true,
```

Comment:
what's the behavior that authTypes false here?

Reply:
UX components won't able to read [this config](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5916/files#diff-5d54489df9aac06050baae247879eddfb5057500423a631187245e7331238e0aR276) if we set it to false.